### PR TITLE
Migrate commons 

### PR DESCRIPTION
**Pull request by nolawi**
Connects #4932 

### Description
Moved all styles from commons as is to local styles

### Acceptance Criteria 
- [ ]  No changes in styles of app

### Testing Plan
1. Bundle install
2. Do a `rake assets:precompile` and check public folder if USWDS fonts are loading in `assets/fonts/uswds/`
screen shot 2018-04-12 at 4 59 04 pm

3. Visit-> Dispatch, Stats/ Styleguide/ Reader /Hearings Prep - test styles against demo

### CODE REVIEW

File: app/assets/stylesheets/_stats.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
+@import 'bourbon';
+@import 'neat';
+@import 'uswds/all';
+@import 'global/fonts';
+
+.cf-stats-container {
```
Comment (nolawi) [inline app/assets/stylesheets/_stats.scss]: added a container here for conflict issues

File: app/assets/stylesheets/_commons.scss [hunk 2]
```diff
@@ -0,0 +1,1472 @@
...
+  .cf-loading-indicator {
+    display: block;
+
+    .using-logo {
+      color: transparent;
+      display: inline-block;
+      width: 40%;
+      height: 90%;
+      position: relative;
+      background: image-url('icons/cf-logo-pill.svg') center no-repeat;
```
Comment (mdbenjam) [inline app/assets/stylesheets/_commons.scss]: do we need to bring in the image as well?

**Comment by nolawi** [comment]
screen shot 2018-05-02 at 9 19 08 am
Updated all icon refs, favi

File: app/assets/stylesheets/_main.scss [hunk 3]
```diff
@@ -600,36 +595,6 @@ select {
   color: $color-gray;
 }
 
-//move to commons if/when accordions are used for any non-React apps
-.usa-accordion-button {
```
Comment (mdbenjam) [inline app/assets/stylesheets/_main.scss]: Why did we remove this?

Comment (mdbenjam) [reply app/assets/stylesheets/_main.scss]: whoops jk saw it was already in commons.

**Pull request merged**